### PR TITLE
Remove compiled files so we can generate them ourselves

### PR DESCRIPTION
**Pull request by rchiodo**
Removing all of the compiled dlls for attach so we can generate them ourselves.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,10 @@ __pycache__/
 
 # C extensions
 *.so
+*.dylib
```
Comment (AdamYoblick) [inline .gitignore:8]: I'm not sure if there are any other files with these extensions in our repo. Should be consider explicitly excluding these files from `src\debugpy\_vendored\pydevd\pydevd_attach_to_process` ?

**Review by AdamYoblick — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (rchiodo) [reply .gitignore:8]: There isn't any but I think we shouldn't ever be checking in dlls/exes. Too opaque.

**Comment by rchiodo** [comment]
Can you just submit this Adam? It's not performing the license test for some reason

**Comment by rchiodo** [comment]
Okay the 3.13 tests all passed locally with the same binaries as built by the PR. Submitting.

**Pull request merged**